README.md

```diff
@@ -107,4 +107,4 @@ or via `conda`:
 conda install -c conda-forge zarr
 ```
 
-For more details, including how to install from source, see the [installation documentation](https://zarr.readthedocs.io/en/stable/#installation).
+For more details, including how to install from source, see the [installation documentation](https://zarr.readthedocs.io/en/latest/installation.html).
```

Comment:
Think we want to keep this as `stable` per ( https://github.com/zarr-developers/zarr-python/issues/1176#issuecomment-1276572422 )

```suggestion
For more details, including how to install from source, see the [installation documentation](https://zarr.readthedocs.io/en/stable/installation.html).
```

cc @joshmoore

Reply:
Definitely. :+1: